### DISCUSSION TITLE
Desoldering the 5V voltage regulator from the Arduino board

### DISCUSSION THREAD

**Discussion by ghost** [sanni/cartreader#122] [Need help]
Probably a silly question but, the Arduino I have has slightly different looking parts than the Arduino in the PCB building video, and I would like to confirm the correct parts to de-solder if that's ok?
There is an AMS1117 3.3 as well as an AMS1117CD 5.0 on the board. The AMS1117 3.3 looks like what is shown in the video, but I feel like the AMS1117CD 5.0 is what I should be removing, as the wiki states "5V voltage regulator" specifically. Thanks for your time and help!

[RobotDyn MegaXPro 2560 R3](https://ae01.alicdn.com/kf/Hae94b4ffdcb8440c9e39f69428bd68a4D/Mega-XPro-2560-R3-extra-86-I-O-CH340C-ATmega2560-16AU-RGB-LED-5V-3-3V.jpg_Q90.jpg_.webp)

[AMS1117CD 5.0](https://lcsc.com/product-detail/Dropout-Regulators-LDO_Advanced-Monolithic-Systems-AMS-AMS1117CD-5-0_C841797.html)

[AMS1117 3.3](https://lcsc.com/product-detail/Dropout-Regulators-LDO_Advanced-Monolithic-Systems-AMS1117-3-3_C6186.html)

**Comment by sanni** [comment]
I have not tested this version but from what I can tell by the pictures it should work the same. Desolder the AMS1117CD 5.0 and the transistor below the green fuse, then connect the two wires, one after the fuse, one to GND.
